welcome-to-the-bitcoin-protocol/ANSWERS.md

```diff
@@ -0,0 +1,31 @@
+# Some Responses to the Discussion Questions
+
+Q: Why did Satoshi put so much stress on the irreversibility of transactions?  
+A: The system of electronic payments made with credit and with the possibility of transaction reversal introduces the need for more trust. It creates a system where the trusted third parties that facilitate payments (banks) are required to mediate disputes, which increases the cost to transact. Some fraud will always occur in such a system, further increasing costs. Bitcoin's creator wanted its payment system to be the digital equivalent of transacting with cash, where payments can't be reversed once sent. Thereby eliminating the need for trusted third parties to mediate disputes and the risk of being defrauded.
+
+Q: What proposed role do SPV nodes play in the bitcoin ecosystem?  
+A: SPV nodes have the ability to verify that a payment has been included in the blockchain, without bearing the costs of running a full node. An SPV node downloads and maintains a copy of the most-work chain's block headers, and can verify that a payment was included in a given block by querying peers for that block's Merkle branch data.
+
+Q: How have show-stopping bugs or disruptions to the network been handled in the past?  
+A: At one time, the community was small enough that Satoshi and early maintainers were able to simply make code changes, entice users to upgrade, and ask miners to do any reorganizing necessary to remedy major bugs, such one where a transaction was introduced in a block that caused an overflow.
+
+Q: In your opinion, what did Satoshi "invent" that was truly innovative?  
+A: One of Satoshi's key inventions was the minting and transferring of money in a non-reversible way by using Proof of Work.
```

Comment:
I think this one is too subjective to answer definitively. I'd like to remove this from the answers.